core/src/main/scala/kafka/tools/DumpLogSegments.scala

```diff
@@ -69,7 +69,7 @@ object DumpLogSegments {
 
     CommandLineUtils.checkRequiredArgs(parser, options, filesOpt)
 
-    val print = if(options.has(printOpt)) true else false
+    val print = if(options.has(printOpt) || options.has(offsetsOpt) || options.has(valueDecoderOpt) || options.has(keyDecoderOpt)) true else false
```

Comment:
@cotedm Sorry, I meant can we replace the `if (condition) true else false` with `condition`? There are a couple similar cases below.